scripts/setup-host.sh

```diff
@@ -27,6 +27,26 @@ check_environment() {
         msg "NOTE: external_address has not been set in ${git_root}/configuration/config.toml. You might need it if your external IP address is different."
         msg
     fi
+
+    local shared_lib_path
```

Comment:
Need to update docs as well:
https://docs.axelar.dev/node/config-node#home-directory
https://docs.axelar.dev/validator/setup/manual

Reply:
https://github.com/axelarnetwork/axelar-docs/pull/759